src/runtime_src/core/pcie/emulation/hw_em/generic_pcie_hal2/shim.cxx

```diff
@@ -22,7 +22,7 @@
 #include <errno.h>
 #include <unistd.h>
 #include <boost/lexical_cast.hpp>
-
+#include "core/common/xclbin_parser.h"
 #include "xcl_perfmon_parameters.h"
 #define SEND_RESP2QDMA() \
     { \
```

Comment:
Outside of this PR...
Just curious about why a macro in this C++ code instead of a function or lambda?

Reply:
its not related to my PR, will work on this as a septate PR.